packages/tools/viewer/src/viewerFactory.ts

```diff
@@ -92,7 +95,11 @@ export async function CreateViewerForCanvas(
         case "WebGPU": {
             // eslint-disable-next-line @typescript-eslint/naming-convention, no-case-declarations
             const { WebGPUEngine } = await import("core/Engines/webgpuEngine");
-            const webGPUEngine = new WebGPUEngine(canvas, options);
+            const webGPUEngine = new WebGPUEngine(canvas, {
+                ...options,
+                enableAllFeatures: defaultCanvasViewerOptions.enableAllFeatures,
+                setMaximumLimits: defaultCanvasViewerOptions.setMaximumLimits,

```

Comment:
This should follow the same pattern as antialias and adaptToDevicePixel ratio in the proxy above. This makes it possible for the options to change dynamically, which the viewer configurator relies on.